api_pagopa.yaml

```diff
@@ -61,30 +58,11 @@ definitions:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v5.0.0/openapi/definitions.yaml#/ProblemJson"
   FiscalCode:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v5.0.0/openapi/definitions.yaml#/FiscalCode"
+  FederatedUser:
```

Comment:
What's the purpose of this new definition?

Reply:
it's not new (it was already defined before this PR), it's the G.C.D. of external (federated) user's properties